### PR TITLE
Updated schema for enrichr datasets

### PR DESCRIPTION
**Pull request by jeevangelista**

### CODE REVIEW

File: meta/entity/draft-1.json [hunk 1]
```diff
@@ -17,7 +29,8 @@
         }
       },
       "required": [
-        "aliases"
+        "aliases",
```
Comment (u8sand) [inline meta/entity/draft-1.json]: If we're still using `aliases` maybe it should become capital like the rest of them. And maybe renamed to Synonyms?

File: meta/signature/draft-1.json [hunk 2]
```diff
@@ -138,33 +174,30 @@
...
         },
         "Gene": {
           "description": "The gene (TODO: validate against official list)",
-          "type": "string"
+          "type": "array",
+          "items": {
+            "type": "object",
+            "properties": {
+              "Name": {
+                "description": "Name of the disease",
```
Comment (u8sand) [inline meta/signature/draft-1.json]: Is this a disease or a gene?

File: meta/signature/draft-1.json [hunk 3]
```diff
@@ -274,7 +290,32 @@
...
+                "required": [
+                  "Original String"
+                ]
+              },
+              {
+                "required": [
+                  "Name"
+                ]
+              }
+            ]
```
Comment (u8sand) [inline meta/signature/draft-1.json]: What makes the Name different from Original String?

File: meta/signature/draft-1.json [hunk 4]
```diff
@@ -318,7 +359,27 @@
         },
         "Small Molecule": {
           "description": "Small molecule used in the study", 
-          "type": "string"
+          "type": "array",
+          "items": {
+            "type": "object",
+            "properties": {
+              "Name": {
+                "description": "Name of the disease",
```
Comment (u8sand) [inline meta/signature/draft-1.json]: Is this a disease or a small molecule?

File: meta/signature/draft-1.json [hunk 5]
```diff
@@ -318,7 +359,27 @@
...
+                "type": "string"
+              },
+              "Accession": {
+                "description": "Ontology accession",
+                "type": "string"
+              },
+              "Concentration": {
+                "description": "External reference for the disease",
```
Comment (u8sand) [inline meta/signature/draft-1.json]: Not sure i understand this description

File: meta/entity/draft-1.json (hunk 1 above)
Comment (jeevangelista) [reply meta/entity/draft-1.json]: fixed 1c2240d9e49e8cbd4343cf2cc8ac551f2af9bf92

File: meta/signature/draft-1.json (hunk 3 above)
Comment (jeevangelista) [reply meta/signature/draft-1.json]: Hmm... original string is the original wording of the metadata on the gmt files (i.e. not checked against an ontology, that's the name). I retained the original string as it contains some other information that might be useful + for validation later. Then again, we are already storing the original gmt metadata string so it is also possible to remove this altogether.

Comment (jeevangelista) [reply meta/signature/draft-1.json]: 6d08aacc5326b0394a565bad2f6ec59b39316f2a

File: meta/signature/perturbation/small-molecule.json [hunk 6]
```diff
@@ -0,0 +1,43 @@
+{
+  "$id": "/@dcic/signature-commons-schema/meta/signature/perturbation/small-moleculß.json",
```
Comment (u8sand) [inline meta/signature/perturbation/small-molecule.json]: You've got a strange character here

File: meta/entity/draft-1.json [hunk 7]
```diff
@@ -8,7 +8,19 @@
...
       "type": "object",
       "properties": {
-        "aliases": {
+        "Name": {
+          "type": "string",
+          "description": "Preferred gene name"
+        },
+        "Taxon id": {
+          "type": "string",
+          "description": "Preferred gene name"
```
Comment (u8sand) [inline meta/entity/draft-1.json]: This description doesn't match the key

File: meta/entity/draft-1.json [hunk 8]
```diff
@@ -8,7 +8,19 @@
     {
       "type": "object",
       "properties": {
-        "aliases": {
+        "Name": {
+          "type": "string",
+          "description": "Preferred gene name"
+        },
+        "Taxon id": {
```
Comment (u8sand) [inline meta/entity/draft-1.json]: `Gene ID`
`Taxon id`

Let's use either `ID` or `id` consistently; perhaps normalize to `ID`

File: meta/signature/perturbation/Phenotype.json [hunk 9]
```diff
@@ -0,0 +1,32 @@
+{
+  "$id": "/@dcic/signature-commons-schema/meta/signature/perturbation/phenotype.json",
```
Comment (u8sand) [inline meta/signature/perturbation/Phenotype.json]: This filename should be lowercase like the other files, currently it is `Phenotype.json`

File: meta/signature/perturbation/disease.json [hunk 10]
```diff
@@ -0,0 +1,35 @@
...
+  "$id": "/@dcic/signature-commons-schema/meta/signature/perturbation/disease.json",
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "allOf": [
+    {
+      "$ref": "/@dcic/signature-commons-schema/core/meta.json"
+    },
+    {
+      "type": "array",
+      "description": "The disease being studied",
+      "items":[
```
Comment (u8sand) [inline meta/signature/perturbation/disease.json]: Space here

File: meta/signature/perturbation/other.json [hunk 11]
```diff
@@ -0,0 +1,35 @@
+{
+  "$id": "/@dcic/signature-commons-schema/meta/signature/perturbation/disease.json",
```
Comment (u8sand) [inline meta/signature/perturbation/other.json]: This `$id` doesn't match the filename

File: meta/signature/perturbation/other.json [hunk 12]
```diff
@@ -0,0 +1,35 @@
...
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "allOf": [
+    {
+      "$ref": "/@dcic/signature-commons-schema/core/meta.json"
+    },
+    {
+      "type": "array",
+      "description": "Other kinds of perturbation",
+      "items":[
```
Comment (u8sand) [inline meta/signature/perturbation/other.json]: Space here

File: meta/signature/perturbation/other.json [hunk 13]
```diff
@@ -0,0 +1,35 @@
...
+        {
+          "type": "object",
+          "properties": {
+            "Name": {
+              "description": "The name of the perturbation",
+              "type": "string"
+            },
+            "Accession" : {
```
Comment (u8sand) [inline meta/signature/perturbation/other.json]: Extra space here

File: .gitignore [hunk 14]
```diff
@@ -1,8 +1,12 @@
...
+.env
+\.DS_Store
+__pycache__/
 build/
 dist/
-node_modules/
\ No newline at end of file
+key_group\.txt
+node_modules/
+
```
Comment (u8sand) [inline .gitignore]: Blank space?

**Pull request merged**